pattern-library/sass/utilities/_afontgarde.scss

```diff
@@ -0,0 +1,57 @@
+/*! afontgarde - v0.1.6 - 2015-03-13
```

Comment:
I commented in a known Sass github issue around being able to import plain old CSS files in a Sass import (https://github.com/sass/sass/issues/279#issuecomment-136926782), FYI.